components/scream/src/physics/p3/p3_main_impl.hpp

```diff
@@ -185,7 +185,7 @@ ::p3_main_part1(
       // prescribe that value
 
       if (do_prescribed_CCN) {
-         nc(k).set(not_drymass, max(nc(k), nccn_prescribed(k)));
+         nc(k).set(not_drymass, max(nc(k)*1.0e6*inv_rho(k), nccn_prescribed(k)));
```

Comment:
@hassanbeydoun , the tests failed, but I think it is because this change doesn't match the F90 change.  Note the F90 change above is

```
nc(k) = max(nc(k),nccn_prescribed(k)*1.0e6*inv_rho(k))
```

There the `nccn_prescribed` is multiplied by the conversion factor, not `nc`

Reply:
Oops! Thanks for pointing this out Aaron. Fixed now.